skrub/_utils.py

```diff
@@ -37,6 +37,14 @@ def __contains__(self, key: Hashable):
         return key in self.cache
 
 
+def combine_LRUDicts(capacity: int, *LRUDicts: LRUDict) -> LRUDict:
+    combined_LRUDict = LRUDict(capacity)
+    for lru_dict in LRUDicts:
+        for key, value in lru_dict.cache.items():
+            combined_LRUDict[key] = value
+    return combined_LRUDict
```

Comment:
Small improvements:
```suggestion
def combine_LRUDicts(capacity: int, *lru_dicts: LRUDict) -> LRUDict:
    combined_lru_dict = LRUDict(capacity)
    for lru_dict in lru_dicts:
        for key, value in lru_dict.cache.items():
            combined_lru_dict[key] = value
    return combined_lru_dict
```
Also the way the function is named does not follow the snake_case convention, it might be better named `combine_lru_dicts`, what do you think?